tmt/steps/provision/__init__.py

```diff
@@ -1471,8 +1471,6 @@ def _run_ansible(
         :param silent: if set, logging of steps taken by this function would be
             reduced.
         """
-        playbook = self._ansible_playbook_path(playbook)
```

Comment:
IIRC, the idea was that `_ansible_playbook_path()` should be fixed rather than removed. It's there to make sure the playbook lives in directories tmt controls, and the side-effect is it produces an absolute path to the playbook which you are unhappy about. Let's fix that, but not by throwing away the check like this. We'll investigate & find something more complicated and uglier :)

Reply:
Your recommendation has been implemented.

Reply:
Thanks!